components/admin_console/billing/billing_subscriptions/index.tsx

```diff
@@ -63,6 +67,46 @@ const BillingSubscriptions: React.FC<Props> = () => {
 
     const [showCreditCardBanner, setShowCreditCardBanner] = useState(true);
 
+    const product = useSelector((state: GlobalState) => {
+        const products = state.entities.cloud.products!;
+        if (!products) {
+            return null;
+        }
+        const keys = Object.keys(products);
+        let product: Product;
+        if (products && subscription) {
+            product = products[subscription?.product_id];
+            if (!product) {
+                keys.forEach((key) => {
+                    if (products[key].name.toLowerCase().includes('professional')) {
+                        product = products[key];
+                    }
+                });
+            }
+            if (product) {
+                return product;
+            }
+        }
+        return products[keys[0]];
+    });
+
+    let subscriptionPlan = 'CLOUD_PROFESSIONAL';
+
+    switch (product?.sku) {
+    case 'cloud-starter':
+        subscriptionPlan = 'CLOUD_STARTER';
+        break;
+    case 'cloud-professional':
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    case 'cloud-enterprise':
+        subscriptionPlan = 'CLOUD_ENTERPRISE';
+        break;
+    default:
+        subscriptionPlan = 'CLOUD_PROFESSIONAL';
+        break;
+    }
```

Comment:
Let's make the SKU's constants, so you can use them everywhere and only have to change one var.